### ISSUE TITLE
Staging the gdas-obs for Marine DA

### ISSUE THREAD

**Issue by apchoiCMD** [NOAA-EMC/global-workflow#2706]
### Target fix directory

NEW

### Type of change

- [ ] Update existing files
- [ ] Remove files
- [X] Add new files to existing directory
- [ ] Add new fix directory

### Any workflow changes needed?

No change needed

### Related issues

https://github.com/NOAA-EMC/GDASApp/issues/1120

### Pickup location

/work/noaa/da/marineda/gfs-marine/data/obs
OR
/scratch1/NCEPDEV/global/glopara/data/experimental_obs

### Size

0

**Comment by WalterKolczynski-NOAA** [comment]
```
ls: cannot access '/work/noaa/da/marineda/gfs-marine/data/obs': Permission denied
```

**Comment by apchoiCMD** [comment]
@WalterKolczynski-NOAA Sorry for your inconvenience- I've changed the permissions so you will be able to access the folders/files

**Comment by WalterKolczynski-NOAA** [comment]
Looks like these need to go in the obs dump rather than a fix directory. Will coordinate with @KateFriedman-NOAA on how to do this.

**Comment by guillaumevernieres** [comment]
> Looks like these need to go in the obs dump rather than a fix directory. Will coordinate with @KateFriedman-NOAA on how to do this.

@WalterKolczynski-NOAA , these are still experimental obs. They currently are here for the gdas:
```/scratch1/NCEPDEV/global/data/experimental_obs/```
We just need to add the gfs obs at the same level.

**Comment by apchoiCMD** [comment]
Updated obs location with Hera as well, please use/copy the gfs obs:
```
/scratch1/NCEPDEV/global/glopara/data/experimental_obs
```

**Comment by apchoiCMD** [comment]
@KateFriedman-NOAA I just wanted to touch base to see if there are any updates on this issue. Always thanks!

**Comment by KateFriedman-NOAA** [comment]
@apchoiCMD Apologies, haven't been able to get to this yet. I've just taken a look at the obs you staged on Orion. I have some questions...would you have time for a short Meet tomorrow between 11am-noon or 1-2pm? Thanks!

**Comment by apchoiCMD** [comment]
@KateFriedman-NOAA 11am-12pm should work for me, thanks for your offer-

**Comment by apchoiCMD** [comment]
Either you can use them at `/scratch1/NCEPDEV/da/common`

**Comment by KateFriedman-NOAA** [comment]
Just spoke offline with @apchoiCMD and discussed the contents of the experimental obs. Given the holiday, I have planned to pull the obs into the experimental_obs folder on Hera next Monday and when @WalterKolczynski-NOAA returns from leave he will sync them to Orion. There is no need currently for the obs on any other supported platforms.

Will update this issue once obs are in place on Hera.

**Comment by KateFriedman-NOAA** [comment]
@apchoiCMD I have begun copying the experimental marine obs into the glopara space on Hera: `/scratch1/NCEPDEV/global/glopara/data/experimental_obs`

I have several streams of rsyncs happening in parallel. It will likely take most of today, possibly longer to copy them all in. Once in, I'll ask that you take a look to make sure they look good on Hera. I'll let you know when the rsyncs complete.

When do you need them on Orion/Hercules?

**Comment by apchoiCMD** [comment]
> @apchoiCMD I have begun copying the experimental marine obs into the glopara space on Hera: `/scratch1/NCEPDEV/global/glopara/data/experimental_obs`
> 
> I have several streams of rsyncs happening in parallel. It will likely take most of today, possibly longer to copy them all in. Once in, I'll ask that you take a look to make sure they look good on Hera. I'll let you know when the rsyncs complete.
> 
> When do you need them on Orion/Hercules?

@KateFriedman-NOAA No worries, we will test the cycle once obs are ready on Orion- Appreciate your update!

**Comment by JessicaMeixner-NOAA** [comment]
> There is no need currently for the obs on any other supported platforms.

@KateFriedman-NOAA  (FYI: @apchoiCMD @guillaumevernieres):  @CatherineThomas-NOAA and I were just talking.  We need these obs on WCOSS2 as well.  Is that possible?

**Comment by KateFriedman-NOAA** [comment]
@apchoiCMD The marine obs are now in place on Hera, see `/scratch1/NCEPDEV/global/glopara/data/experimental_obs` and let me know if I'm missing any dates. @WalterKolczynski-NOAA will sync them to Orion/Hercules when he's back next week.

@JessicaMeixner-NOAA Yeah, I can sync them to WCOSS2 as well. I have made a note to start that sync tomorrow.

**Comment by JessicaMeixner-NOAA** [comment]
Thank you @KateFriedman-NOAA !

**Comment by apchoiCMD** [comment]
More thanks from me @KateFriedman-NOAA

**Comment by KateFriedman-NOAA** [comment]
@JessicaMeixner-NOAA @apchoiCMD The experimental marine obs are now also on both WCOSS2s. I placed them in a separate folder within the global dump archive (GDA) space since the GDA has it's own separate quota on WCOSS2 and plenty of space right now. I made a symlink from the data folder to the GDA folder so it's similar to the placement on the other platforms. Example:
```
WCOSS2 (PRIMARYSYS-D) data> pwd
/lfs/h2/emc/global/noscrub/emc.global/data
WCOSS2 (PRIMARYSYS-D) data> ll | grep experimental_obs
lrwxrwxrwx  1 emc.global global      46 Jul 15 13:08 experimental_obs -> /lfs/h2/emc/dump/noscrub/dump/experimental_obs
```

Once @WalterKolczynski-NOAA is able to copy these experimental obs to Orion we can close this issue.
